src/test/java/com/mate/fruitshop/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package com.mate.fruitshop.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.mate.fruitshop.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.After;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static final String SOME_TEXT = "12345qwerty";
+    private static final String CORRECT_FILE_NAME = "src/test/resources/testReport.csv";
+    private static WriterService writer = new WriterServiceImpl();
+
+    @After
+    public void tearDown() throws Exception {
+        Path testFile = Path.of(CORRECT_FILE_NAME);
+        if (Files.exists(testFile)) {
+            Files.delete(testFile);
+        }
+    }
+
+    @Test
+    public void write_Empty_Ok() {
+        writer.write("", CORRECT_FILE_NAME);
+        assertTrue(Files.exists(Path.of(CORRECT_FILE_NAME)));

```

Comment:
assertTrue(Files.exists(Path.of(CORRECT_FILE_NAME))) isn't necessary because if file doesn't exist test won't passed in any case